.github/workflows/post_release_version_bump.yml

```diff
@@ -102,6 +101,25 @@ jobs:
           sed -i 's/python:v.*"/python:v'$VERSION'"/' .github/workflows/daily_scan.yml
           git add aws-opentelemetry-distro/src/amazon/opentelemetry/distro/version.py
           git add .github/workflows/daily_scan.yml
+  
+      - name: Append latest release checksum to release-build-metadata.json
+        run: |
+          ARTIFACT_NAME="aws_opentelemetry_distro-${{ github.event.inputs.version }}-py3-none-any.whl"
+          curl -L -o $ARTIFACT_NAME https://github.com/aws-observability/aws-otel-python-instrumentation/releases/download/v${{ github.event.inputs.version }}/$ARTIFACT_NAME
+
+          CHECKSUM=$(shasum -a 256 $ARTIFACT_NAME | awk '{ print $1 }')
+
+          FILE="release-build-metadata.json"
+          UPDATED_JSON=$(jq --arg version "${{ env.VERSION }}" \
```

Comment:
use either `github.event.inputs.version` or `env.VERSION` throughout. Mixing them is confusing and prone to bugs.

Reply:
Ack